scripts/kafka_base_dir.sh

```diff
@@ -0,0 +1,6 @@
+if [ "$TRAVIS" == "true" ]
```

Comment:
The hashbang is there. And the double-equals is a typo indeed. '[[' is hardly used as far as I can tell, would probably raise more eyebrows than be helpful in making this more readable.